sample/android-app/src/main/java/com/icerockdev/MainActivity.kt

```diff
@@ -33,6 +34,12 @@ class MainActivity : AppCompatActivity() {
 
         stringDescTextView.text = Testing.getStringDesc().toString(context = this)
         stringDescTextView.typeface = Testing.getFont2().getTypeface(context = this)
+
+        listOf(
+            Testing.getTextFile().readText(context = this),
+            Testing.getJsonFile().readText(context = this),
+            Testing.getNestedJsonFile().readText(context = this)
```

Comment:
Is need to handle exceptions here when working with files?

Reply:
this files is contained in resources and plugin generate safe code for access to this resources, so i not see any exceptions what really can be throw